charts/newrelic-infrastructure-v3/templates/ksm/deployment.yaml

```diff
@@ -127,6 +127,11 @@ spec:
             {{- end }}
             - name: NRIA_IS_FORWARD_ONLY
               value: "true"
+            - name: NRIA_OVERRIDE_HOSTNAME_SHORT
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
```

Comment:
Is this duplicated? Should be there already: https://github.com/newrelic/helm-charts/blob/c9136c183bfbf27da9ff0ca350995b8c2b1e3a63/charts/newrelic-infrastructure-v3/templates/ksm/deployment.yaml#L87-L91

Reply:
![](https://c.tenor.com/eiYNWS2e6msAAAAM/youre-right-breaking-bad.gif)